### PR TITLE
add re note

### PR DESCRIPTION
**Pull request by okantirpanci**

### CODE REVIEW

**Review by FutureBaseEditor — APPROVED** [review]

**Review by mfdsoftdev — CHANGES_REQUESTED** [review]
the tone of the note hasn't been fully met, can you change it please?

**Review by mfdsoftdev — DISMISSED** [review]

File: Program.cs [hunk 1]
```diff
@@ -4,6 +4,9 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            int re = 293;
```
Comment (FutureBaseEditor) [inline Program.cs:7]: Bu kod olmamaış!!!!

**Review by RecepPekdas — APPROVED** [review]

**Review by FutureBaseEditor — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (FutureBaseEditor) [inline Program.cs:7]: akassasa

**Review by RecepPekdas — APPROVED** [review]

**Review by ZeynepColakoglu — APPROVED** [review]

**Review by AlperVatansever — DISMISSED** [review]

**Review by gultan7 — DISMISSED** [review]

**Review by Ali1988Baba — DISMISSED** [review]

**Review by Ali1988Baba — DISMISSED** [review]
it's good

**Review by mfdsoftdev — DISMISSED** [review]

**Review by AlperVatansever — APPROVED** [review]

**Review by Ali1988Baba — APPROVED** [review]
okk

**Pull request merged**